### ISSUE TITLE
Error on loading module

### ISSUE THREAD

**Issue by z4rcher** [tobi-wan-kenobi/bumblebee-status#473]
#### Summary
Affected module: getcrypto

#### Description:
After upgrading to Fedora 30, I'm getting the following error:
`No handlers could be found for logger "bumblebee.engine"`
`Traceback (most recent call last):`
```
File "/opt/bumblebee-status/bumblebee-status", line 80, in <module>
    main()
  File "/opt/bumblebee-status/bumblebee-status", line 51, in main
    theme=theme,
  File "/opt/bumblebee-status/bumblebee/engine.py", line 142, in __init__
    self.load_modules(config.modules())
  File "/opt/bumblebee-status/bumblebee/engine.py", line 208, in load_modules
    mod = self._load_module(module["module"], module["name"])
  File "/opt/bumblebee-status/bumblebee/engine.py", line 251, in _load_module
    raise bumblebee.error.ModuleLoadError("unable to load module {}: {}".format(module_name, str(err)))
bumblebee.error.ModuleLoadError: unable to load module getcrypto: cannot import name _normalize_host
```
#### How to reproduce

Running the command `/opt/bumblebee-status/bumblebee-status -m getcrypto` under Fedora 30

**Comment by tobi-wan-kenobi** [comment]
Hm, that's a weird issue.

Unfortunately, not able to reproduce on Arch.

Are you using git or one of the release versions?

**Comment by z4rcher** [comment]
Yeah, I cloned the repo with git

**Comment by tobi-wan-kenobi** [comment]
I suspect it might have something to do with the versions of urllib and requests, but I haven't been able to reproc until now, unfortunately :(

Do you have similar issues with other modules that use requests (weather, stock)?

**Comment by z4rcher** [comment]
> Do you have similar issues with other modules that use requests (weather, stock)?

Indeed I tested them as well and they all threw the same error. Anyways, I finally fixed it by switching from Python 2.7 to Python 3.7 and installing psutil and netifaces modules. 

What's curious is that while both Python versions are using requests 2.22.0 and urllib3 1.25.7 only Python 3.7 doesn't throw an error.

**Comment by tobi-wan-kenobi** [comment]
That's really strange.

I'll still close the issue for now, as it's resolved.

Thanks for finding a workaround :)
